tests/fibshow_input/appl_db.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please add the unit-test to have:

1. Empty nexthop
2. Empty ifname
3. IPv6 cases
4. Route table when VRF is present. E.g: `ROUTE_TABLE:VRF-Red:192.224.40.128/25`

Reply:
done

Reply:
Sorry, I forget to save this case

Reply:
done